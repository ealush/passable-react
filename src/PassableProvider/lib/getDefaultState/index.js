export default function getDefaultState() {
    return {
        errors: {},
        warnings: {},
        warningCount: 0,
        errorCount: 0,
        fields: {}
    }
};