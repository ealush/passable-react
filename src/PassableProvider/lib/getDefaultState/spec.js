import getDefaultState from './index';

describe('Test getDefaultState function', () => {
    it('should return correct default state', () => {
        expect(getDefaultState()).toMatchSnapshot();
    })
});