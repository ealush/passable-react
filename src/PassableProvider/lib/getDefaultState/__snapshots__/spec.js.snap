// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test getDefaultState function should return correct default state 1`] = `
Object {
  "errorCount": 0,
  "errors": Object {},
  "fields": Object {},
  "warningCount": 0,
  "warnings": Object {},
}
`;
